prometheus_kafka_consumer_group_exporter/__init__.py

```diff
@@ -10,6 +11,10 @@
 
 METRIC_PREFIX = 'kafka_consumer_group_'
 
+sasl_enabled = os.environ.get('SASL_ENABLED', 'false')
```

Comment:
You could eliminate this, and use `None` for the default value on username and password, then just infer this using `sasl_username is not None and sasl_password is not None`